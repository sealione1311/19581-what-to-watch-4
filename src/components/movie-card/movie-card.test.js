import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from "../movie-card/movie-card.jsx";

const film = {
  id: 1,
  name: `Bohemian Rhapsody`,
  src: `img/bohemian-rhapsody.jpg`
};

describe(`Render MovieCard`, () => {
  it(`Render MovieCard`, () => {
    const tree = renderer.create(
        <MovieCard
          movie = {film}
          onTitleClick = {() => {}}
          onCardHover = {() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
