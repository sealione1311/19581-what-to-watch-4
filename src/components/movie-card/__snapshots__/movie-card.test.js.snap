// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render MovieCard Render MovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={1}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Bohemian Rhapsody"
      height="175"
      src="img/bohemian-rhapsody.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Bohemian Rhapsody
    </a>
  </h3>
</article>
`;
